src/sentry/analytics/events/relocation_created.py

```diff
@@ -0,0 +1,14 @@
+from sentry import analytics
+
+
+class RelocationCreatedEvent(analytics.Event):
+    type = "relocation.created"
+
+    attributes = (
+        analytics.Attribute("user_id"),
```

Comment:
What is this, and how does it differ from `owner_id`? If it's meant to be `creator_id`, we should call it that.